src/components/navBar/navBar.js

```diff
@@ -240,7 +245,9 @@ MdNavBarController.prototype._updateInkBarStyles = function(tab, newIndex, oldIn
   var left = tabEl.offsetLeft;
 
   this._inkbar.toggleClass('_md-left', newIndex < oldIndex)
-      .toggleClass('_md-right', newIndex > oldIndex);
+      .toggleClass('_md-right', newIndex > oldIndex)
+      .toggleClass('md-button', newIndex === -1);
```

Comment:
For some reason there is a md-button class that is adding the border-bottom color

<img width="130" alt="screen shot 2016-06-11 at 10 09 21 am" src="https://cloud.githubusercontent.com/assets/12898908/15986104/974674c0-2fbc-11e6-8f12-89b2800c3dd8.png">

If I don't toggle that class the ink from the text disappear but the border bottom one persists.

<img width="121" alt="screen shot 2016-06-11 at 10 10 45 am" src="https://cloud.githubusercontent.com/assets/12898908/15986107/c84b697c-2fbc-11e6-9697-7cd4e523ffa4.png">

If I do toggle the class, the border-bottom color also disappears

<img width="419" alt="screen shot 2016-06-11 at 10 11 41 am" src="https://cloud.githubusercontent.com/assets/12898908/15986111/e9528862-2fbc-11e6-89b3-8a57ce34d00c.png">